OneCommunityGlobal/HGNRest
# Abi map locations test suite

# Description
Unit and integration tests for the map locations routes and controller

## Related PRS (if any):
N/A
…

## Main changes explained:
- Unit tests for map locations controller
- Integration tests for map locations router
…

## How to test:
1. check into current branch
2. do `npm install` 
3. run `npm run test mapLocations`
4. Verify all tests pass

## Screenshots or videos of changes:
![Screenshot 2024-07-20 at 12 42 58 AM](https://github.com/user-attachments/assets/c66c8dac-7494-4d46-a7f0-c0122b84cd9b)

## Note:
Include the information the reviewers need to know.
